### ISSUE TITLE
False positive in ModifierHeightWithText rule

### ISSUE THREAD

**Issue by dimsuz** [appKODE/detekt-rules-compose#25]
This piece of code 

```kotlin
Row(
  modifier = Modifier
    .height(IntrinsicSize.Min),
) {
  Text(
    modifier = Modifier
      .fillMaxHeight(),
    text = "Hello world",
  )
}
```
results in an error:

> Composable uses "height" modifier and contains a Text child. Use heightIn(min = N.dp) instead [ModifierHeightWithText]

which is incorrect as this doesn't make sense for `IntrinsicSize` overload of `height`.

Exception is needed for this specific usage.
